main.go

```diff
@@ -388,6 +404,9 @@ func loadConfig(filename string, queryUserData bool) (*Config, error) {
 		TasksDir:                       defaultTasksDir(),
 		WorkerGroup:                    "test-worker-group",
 		WorkerTypeMetadata:             map[string]interface{}{},
+		Project:                        "",
```

Comment:
I prefer either:

a) not hitting statsum / sentry if a value isn't defined (which is probably what we want for dev environments)
b) using a default like `test-monitor` such that we have a defined bucket for assigning scopes (e.g. the role for running tests includes the scope to use `test-monitor` in the same way it grants the use of `test-provisioner` etc).